### DISCUSSION TITLE
Algorithm for Pure Upscaling Only?

### DISCUSSION THREAD

**Discussion by MicHaeL-MonStaR** [upscayl/upscayl#366] [Q&A] [answered]
This might seem a weird question, but do any of the algorithms (or is there any out there) that upscales (and reconstructs) images in terms of resolution and quality only?

What I mean is, every algorithm (or "type") seems to behave differently in terms of manipulating the image. - On top of that, it also very much depends on what the nature of the image you're trying to upscale is like. As one algorithm might do weird stuff to the appearance of humans or characters in a photo, such as smooth our their skin and bulge their eyes, another might not do that as much. And where some algorithms might work better for more simplistic graphics, those might also wipe out things like details and intentional noise in other images. - And I've also noticed changes in brightness and coloration.

So it's very hit or miss, and while the literal upscaling in resolution is quite impressive, it often just destroys the original image, looking like an actual weird AI-abomination, and doesn't actually improve the quality of the image either.
I've actually tried things like upscaling an image by the default 4x, then reduced thát image back down to the original's size (which should make an image sharper), and then compare them. But it often results in no difference, and sometimes even slightly worse, plus the removal of detail and change in color and/or lighting.

Is there any algorithm that just leaves things like details and proportions alone, and thus purely upscales the size and quality?
Because for me it's mostly for cases where the source-image might be of high quality, but doesn't quite cut it in terms of resolution for certain applications, such as printing. - Obviously I don't want the original to be messed with in terms of content.

**Comment by aaronliu0130** [comment, marked as answer]
That's not something models can control. All models do is "this part of the image gets turned into this part of the upscaled image, so for every that kind of part of the image I'll turn it into that!". As higher quality often causes smoothening and decrease in noise, and sometimes models will recognize something as something else, there is no model that won't remove noise for now.

Also, somebody called the fake version of Ultrasharp we accidently included "consistent"; they may have meant a similar thing. You can download it from https://github.com/upscayl/custom-models.
